charging.py

```diff
@@ -0,0 +1,38 @@
+def ranges_of_current(current_samples):
+    current_samples_unique = [*set(current_samples)]
+    samples_count = len(current_samples)
+    unique_samples_count = len(current_samples_unique)
+    length = 1
+    list_of_ranges = []
+    count_of_values_in_range = []
+	
+    current_samples.sort()
+    for i in range (1, unique_samples_count + 1):
+        count = 0
+        if (i == unique_samples_count or current_samples_unique[i] - current_samples_unique[i - 1] != 1):
+            if (length == 1):
+                list_of_ranges.append(str(current_samples_unique[i - length]))
+                count_of_values_in_range.append(1)
+            else:
+                temp = (str(current_samples_unique[i - length]) + " - " + str(current_samples_unique[i - 1]))
+                list_of_ranges.append(temp)
+                for j in range (samples_count):
```

Comment:
This `for` and its `if` could be separated into a function like `countConsecutiveSamplesInRange`